src/js/burials/containers/ConfirmationPage.jsx

```diff
@@ -32,10 +34,13 @@ class ConfirmationPage extends React.Component {
 
   render() {
     const form = this.props.form;
-    // const response = this.props.form.submission.response
-    //   ? this.props.form.submission.response.attributes
-    //   : {};
-    const name = form.data.claimantFullName;
+    const response = this.props.form.submission.response
```

Comment:
One thing I did on the hca confirmation page is make sure that if you refresh this page, it loads without a JS error. We would get a lot of those errors logged in Sentry from people doing that.